contest/contest_info.py

```diff
@@ -43,7 +45,7 @@ def get_total_testcases(problem):
     testcases = Testcase.objects.filter(problem = problem)
     return testcases.count()
 
-def get_contestant_problem_submission_list(contest,contestant,problem):
+def get_contestant_problem_submission_list(contest, contestant, problem):
     return Submission.objects.filter(problem = problem, submit_time__lte = contest.end_time,
         submit_time__gte = contest.start_time,user = contestant.user).order_by('submit_time')
```

Comment:
This function is now only used in scoreboard.
And there is a problem if sorted by '''-submit_time'''.
Since we need to check if user has passed the problem, 
so we have to check from oldest to the most recent submission.
And that's the reason why we can use '''-submit_time'''.